### PR TITLE
Update clan-chat-warnings to 1.3.1

### PR DESCRIPTION
**Pull request by JorVaCoding**
Adds the ability to pull a list from a configurable url

### CODE REVIEW

**Review by Nightfirecat — APPROVED** [review]

File: plugins/clan-chat-warnings [hunk 1]
```diff
@@ -1,3 +1,4 @@
 repository=https://github.com/MarbleTurtle/ClanChatWarnings.git
-commit=e25ea129a5e14341f75d9631d7b4d2a5234d5153
+commit=26c978ef030598097335a3835a7a627213fd735e
 authors=MarbleTurtle,JorVaCoding
+warning=This plugin can fetch text from 3rd party websites not controlled or verified by the RuneLite Developers. This setting is optional, and only accesses the user-inputed site.
```
[the remark this answers is not in the document]
Comment (JorVaCoding) [reply plugins/clan-chat-warnings:4]: I added the warning to the config, but would like to keep it on new installations as well.

[the remark this answers is not in the document]
Comment (Nightfirecat) [reply plugins/clan-chat-warnings:4]: [I'm specifically talking about `ConfigItem#warning()` just so you're aware](http://static.runelite.net/api/runelite-client/net/runelite/client/config/ConfigItem.html#warning()). You can see this in effect for the "Compute Shaders" config of the GPU plugin, or the "Enable custom window chrome" or "Enable tray icon" settings of RuneLite configuration.

[the remark this answers is not in the document]
Comment (Bram91) [reply plugins/clan-chat-warnings:4]: Please use the warning field like nfc mentioned, an example can be found here: [Compute shader warning](https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPluginConfig.java#L132)

[the remark this answers is not in the document]
Comment (JorVaCoding) [reply plugins/clan-chat-warnings:4]: I pushed a change to use the proper warning implementation

**Pull request merged**